resources/neumann-testnet-parachain.yaml

```diff
@@ -0,0 +1,67 @@
+networkSpec:
+    displayName: neumann-parachain-testnet-jzia5
+    name: neumann-parachain-testnet-jzia5
+    protocol: polkadot-parachain
+    imageRepository: oaknetwork/neumann
+    imageVersion: 1.2.4-rc2
+    config:
+      nodeTypes:
+        full:
+          args:
+            - key: "--force-authoring"
+            - key: --chain
+              file: "neumann.json"
+            - key: --parachain-id
+              value: 2000
+            - key: --
+            - key: --execution
+              value: wasm
+            - key: --chain
+              file: "rococo-testnet.json"
+            - key: --bootnodes # Relay chain p2p address
+              value: /dns4/node-6896992096614604800-0.p2p.onfinality.io/tcp/23160/ws/p2p/12D3KooWM1vTue78y2ZjxTjZKgYEPR8gSNFtTda3KETgp3WAhCxK
+        collator:
+          args:
+            - key: "--force-authoring"
+            - key: --chain
+              file: "neumann.json"
+            - key: --parachain-id
```

Comment:
i think youi meant to remove this

Reply:
ahh yes yes, thank you. thanks for the catch haha. god knows we don't want to ask OnF this thing again